src/controller/age.controller.js

```diff
@@ -0,0 +1,37 @@
+const { Router } = require("express");
+const router = Router();
+/**
+ * @description Calculate user age
+ * @returns {number}
+ */
+
+router.get("/howold", async (req, res) => {
+  const { dob } = req.query;
+
+  if (!dob || dob == "") {
+    return res.status(400).json({
+      error: "Please provide a date of birth",
+    });
+  }
+
+   const today = new Date();
+  const birthDate = new Date(dob);
+   if (birthDate == "Invalid Date") {
+    return res.status(400).json({ error: "Only date format yyyy-mm-dd is allowed" });
+   }
+  
+   const age = today.getFullYear() - birthDate.getFullYear();
+   if (age < 0) {
+     return res.status(400).json({
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9973834)
```suggestion
    return res.status(400).json({
```